### PR TITLE
Add Pepsi error message when battery state is not available

### LINKED ISSUE #105: Missing error message in pepsi when battery state is not available via aliveness
When using `pepsi aliveness`, there is only an output for the battery if it is discharging or its charging level is below 95%. 
The battery status is retrieved from HuLA and might be unavailable. In this case, `pepsi aliveness` should also output a message, e.g. `󰁽 ?` (this is the same icon as for the discharging battery, if your browser can't display it).

The function `print_summary` in the file `tools/pepsi/src/aliveness.rs` has to be changed for implementing this.

### PR DESCRIPTION
**Pull request by JonasHolz**
## Introduced Changes

Add Pepsi error message when battery state is not available

Fixes #105

## ToDo / Known Issues

None

## Ideas for Next Iterations (Not This PR)

None

## How to Test

Run pepsi aliveness with a robot, which has over 95% charge or without running Hula.

### CODE REVIEW

File: tools/pepsi/src/aliveness.rs [hunk 1]
```diff
@@ -82,10 +83,14 @@ fn print_summary(states: &AlivenessList) {
...
-
                 output.push_str(&format!("{icon} {charge}%{:SPACING$}", ""))
             }
-        }
+        } else {
+            output.push_str(&format!(
+                "{DISCHARGING_ICON}{UNKNOWN_CHARGE}{:SPACING$}",
+                ""
+            ))
+        };
```
Comment (julianschuler) [inline tools/pepsi/src/aliveness.rs:93]:
```suggestion
        }
```

File: tools/pepsi/src/aliveness.rs [hunk 2]
```diff
@@ -82,10 +83,14 @@ fn print_summary(states: &AlivenessList) {
                 } else {
                     DISCHARGING_ICON
                 };
-
                 output.push_str(&format!("{icon} {charge}%{:SPACING$}", ""))
             }
-        }
+        } else {
+            output.push_str(&format!(
+                "{DISCHARGING_ICON}{UNKNOWN_CHARGE}{:SPACING$}",
```
Comment (julianschuler) [inline tools/pepsi/src/aliveness.rs:90]: Consider merging `{DISCHARGING_ICON}{UNKNOWN_CHARGE}` into a single constant with the same content, e.g. `{UNKNOWN_CHARGE_ICON}`. This makes changing this one independent of changing the discharging icon.

**Review by julianschuler — CHANGES_REQUESTED** [review]
Thanks for your contribution! I only have some minor remarks, feel to implement or discuss.

File: tools/pepsi/src/aliveness.rs (hunk 1 above)
Comment (JonasHolz) [reply tools/pepsi/src/aliveness.rs]: Done!

File: tools/pepsi/src/aliveness.rs (hunk 2 above)
Comment (JonasHolz) [reply tools/pepsi/src/aliveness.rs]: I merged them together.

**Review by julianschuler — APPROVED** [review]
LGTM!

**Pull request merged**